### PR TITLE
Feature : Add counter [fixes #27]

### LINKED ISSUE #27: [Feature] Add counter
**Is your feature request related to a problem? Please describe.**
Right now the button doesn't do any action.

**Describe the solution you'd like**
Add a counter bellow the button that increments on every click on the button.
The font family should be the same with the one on the button.
The state of the counter should be in App.js, not inside the button component.

### PR DESCRIPTION
**Pull request by najeebkp**
Feature : Add counter [fixes #27]

## Description
Created a counter component, which will increase when button click happens.

[fixes #27]

Closes #27

## How Has This Been Tested?
Manually tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes. They do not dramatically decrease the test coverage metric
- [x] All new and existing tests passed.
- [x] I give my permission for my code to be used in this project under this license and any future license terms

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
+  const handleButtonClick = () => {
+    setCount(count + 1);
+  };
+  return (
+    <div className="App">
+      <div className="buttonContainer">
+        <ComicButton
+          buttonName="BUTTON"
+          hoverBtnName="HOVER"
+          focusBtnName="FOCUS"
```
Comment (eduardconstantin) [inline src/App.tsx]: Please remove the focus prop

File: src/components/ComicButton/ComicButton.tsx [hunk 2]
```diff
@@ -4,16 +4,22 @@ import { buttonAnim, clickAnim } from "./ComicButton.anim";
...
 
 const randomNo = (min: number, max: number) => {
   return min + Math.floor(Math.random() * (max - min));
 };
 
-export default function ComicButton({ buttonName, hoverBtnName, focusBtnName }: ComicButtonProps) {
+export default function ComicButton({
+  buttonName,
+  hoverBtnName,
+  focusBtnName,
```
Comment (eduardconstantin) [inline src/components/ComicButton/ComicButton.tsx]: remove the prop

File: src/components/ComicButton/ComicButton.tsx [hunk 3]
```diff
@@ -4,16 +4,22 @@ import { buttonAnim, clickAnim } from "./ComicButton.anim";
 import style from "./ComicButton.module.css";
 
 interface ComicButtonProps {
-  buttonName: string,
-  hoverBtnName: string,
-  focusBtnName: string
+  buttonName: string;
+  hoverBtnName: string;
+  focusBtnName: string;
```
Comment (eduardconstantin) [inline src/components/ComicButton/ComicButton.tsx]: Remove focus

File: src/App.tsx [hunk 4]
```diff
@@ -1,11 +1,23 @@
-import ComicButton from './components/ComicButton/ComicButton';
+import React from "react";
```
Comment (eduardconstantin) [inline src/App.tsx]:
```suggestion
import React, { useState } from "react";
```

File: src/App.tsx [hunk 5]
```diff
@@ -1,11 +1,23 @@
...
 
 export default function App() {
-	return (
-		<div className='App'>
-			<div className='buttonContainer'>
-				<ComicButton buttonName='BUTTON' hoverBtnName='HOVER' focusBtnName='FOCUS' />
-			</div>
-		</div>
-	);
+  const [count, setCount] = React.useState<number>(0);
```
Comment (eduardconstantin) [inline src/App.tsx]:
```suggestion
  const [count, setCount] = useState<number>(0);
```

File: src/components/Counter/Counter.tsx [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import style from "./Counter.module.css";
+
+interface CounterProps {
+  count: number;
+}
+
+function Counter({ count }: CounterProps) {
+  return (
+    <>
+      <div className={style.counter}>Count : {count}</div>
```
Comment (eduardconstantin) [inline src/components/Counter/Counter.tsx]:
```suggestion
      <div className={style.counter}>Count - {count}</div>
```

**Review by eduardconstantin — CHANGES_REQUESTED** [review]

**Comment by najeebkp** [comment]
Changes made!

**Review by eduardconstantin — APPROVED** [review]

**Comment by eduardconstantin** [comment]
Thank you! Don't forget to star the repo.

**Comment by eduardconstantin** [comment]
@all-contributors please add @najeebkp for code

**Pull request merged**